maven-plugin-tools-generators/src/main/resources/help-class-source.vm

```diff
@@ -144,7 +144,7 @@ public class HelpMojo
                 }
                 catch ( IOException e )
                 {
-                    throw new MojoExecutionException( e.getMessage(), e );
+                    getLog().warn( "Could not close " + PLUGIN_HELP_PATH + ": " + e.getMessage() );
```

Comment:
We should use a  try-with-resources block, the exception will be eventually added as 'suppressed'

Reply:
@eolivelli 
If @gaul would fix all Java 7 features then we should create a new Jira ticket as we did it in other projects.

Reply:
I would like to let maven plugins use error-prone with the fewest suppressions.  Moving the entire project to Java 7 is one way to do this although I don't know if that aligns with the project's goals.

Reply:
It perfectly fits to our goals. ;-)
We set the Javac version 1.7 and we were updating the code to J7.
Later we will do the same for J8 - inner classes/Lambda, I/O, loops with method chaining, use of the class `Optional`, etc.

Reply:
Done.